Source/Krypton Components/Krypton.Ribbon/View Layout/ViewLayoutRibbonGroupCluster.cs

```diff
@@ -76,7 +76,7 @@ private class ViewToSize : Dictionary<ViewBase, Size> { }
             _endSepVisible = false;
 
             // Create palette used to supply a width to a border edge view
-            PaletteBorderEdgeRedirect borderEdgeRedirect = new(_ribbon.StateCommon.RibbonGroupClusterButton.Border, needPaint);
+            PaletteBorderEdgeRedirect? borderEdgeRedirect = new(_ribbon.StateCommon.RibbonGroupClusterButton.Border, needPaint);
```

Comment:
These should not be nullable, as they are being `new`ed immediately

Reply:
Fixed in latest commit